deployer/templates/fluentd.yaml

```diff
@@ -159,6 +159,10 @@ objects:
               value: ${USE_MUX_CLIENT}
             - name: "MUX_ALLOW_EXTERNAL"
               value: ${MUX_ALLOW_EXTERNAL}
+            - name: "BUFFER_QUEUE_LIMIT"
```

Comment:
the change needs to be added to https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging/templates/fluentd.j2 since the deployer is deprecated and no longer supported.